Meadowlark-js/package.json

```diff
@@ -46,10 +46,11 @@
     "test:lint": "npm run test:lint:ts && npm run test:lint:eslint",
     "test:lint:eslint": "eslint --max-warnings 0 --ext .js,.ts .",
     "test:lint:ts": "tsc -p . --noEmit",
-    "test:unit": "cross-env NODE_OPTIONS=--max-old-space-size=4096 LOG_LEVEL=warn npx jest --testPathIgnorePatterns=\"integration|e2e\"",
-    "test:integration": "cross-env NODE_OPTIONS=--max-old-space-size=4096 LOG_LEVEL=warn npx jest integration --runInBand",
-    "test:unit:coverage": "rimraf coverage/ && npm run test:unit -- --collectCoverage",
-    "test:e2e": "npm run setup:e2e && npm run run:e2e",
+    "test:unit": "cross-env NODE_OPTIONS=--max-old-space-size=4096 LOG_LEVEL=warn jest --projects tests.config/unit",
+    "test:integration": "cross-env NODE_OPTIONS=--max-old-space-size=4096 LOG_LEVEL=warn jest --projects tests.config/integration --runInBand",
+    "test:unit:coverage": "rimraf coverage.unit/ && npm run test:unit -- --collectCoverage",
+    "test:integration:coverage": "rimraf coverage.integration/ && npm run test:integration -- --collectCoverage",
+    "test:e2e": "npm run setup:e2e && jest --projects tests.config/e2e",
```

Comment:
Because of this change, we can now remove line 55 since it doesn't apply anymore